source/guides/guides/test-retries.md

```diff
@@ -0,0 +1,185 @@
+
+- Retried up to two times (for a total of three attempts) in `cypress run` mode
+- Run normally with no retries in `cypress open` mode
+
+The reason for this distinction is to avoid adding unnecessary 
+
+ retried up to two times (for a total of three attempts) in  with no additional configuration needed. When developing locally with Cypress' Open Mode, test retries is disabled by default to prevent blocking developers with unnecessary retries.
+
+If you need more information on how to migrate to 5.x, please see our official migration guide here.
+
+### Global Configuration
+
+If you need to change the number of attempts being made across your entire test suite for both `cypress run` and `cypress open`, you can configure this in your `cypress.json` by defining the `retries` property and giving it the desired number.
+
+```jsx
+{
+  "retries": 1
+}
+```
+
+However, in the event you need to define unique retry attempts for the different modes of Cypress, you can pass the `retries` option an object with the following options:
+
+- `runMode` allows you to define the number of test retries when running `cypress run`
+- `openMode` allows you to define the number of test retries when running `cypress open`
+
+```jsx
+{
+  "retries": {
+    // Configure retries for `cypress run` 
+    // Default is 2
+    "runMode": 1,
+    // Configure retries for `cypress open`
+    // Default is 0
+    "openMode": 3
+  }
+}
+```
+
+### Custom Configurations
+
+#### Individual Test(s)
+
+In the event a specific test requires a custom number of test retries configured, this can be achieved by using the test's local configuration option.
+
+```jsx
+// Customizing retries for an individual test
+describe('User sign-up and login', () => {
+  // `it` test block with no custom configuration
+  it('should redirect unauthenticated user to sign-in page', () => {
+    ...
+  })
+
+  // `it` test block with custom configuration
+  it('allows user to login', { 
+    retries: {
+      runMode: 3,
+      openMode: 2
+    }
+  }, () => {
+    ...
+  })
+})
+
+```
+
+#### Test Suite(s)
+
+If you want a suite of tests to re-run in requires a custom number of test retries configured, this can be achieved by using the test's local configuration option
+
+```jsx
+// Customizing retries for a suite of tests
+describe('User bank accounts', {
+  retries: {
+    runMode: 3,
+    openMode: 1,
+  }
+}, () => {
+  // Individual tests will be run normally
+  it('allows a user to view their transactions, () => {
+    ...
+  }
+
+  it('allows a user to edit their transactions, () => {
+    ...
+  }
+})
+
+```
+
+You can find more information about custom configurations here: {% url "Test Configuration" configuration#Test-Configuration %}
+
+## Assets
+
+With test retries, Cypress will now generate any assets (i.e., screenshots, video recordings, etc.) per each test attempt. The assets will be properly labeled with the attempt number appended at the end of each asset.
```

Comment:
I don't really think there is an etc here is there? What is there beyond screenshots and videos? If there is something, I would be more descriptive. 

```suggestion
With test retries, Cypress will now generate any screenshots and videos per each test attempt. These will be properly labeled with the attempt number appended at the end of each filename.
```